PendingReleaseNotes.md

```diff
@@ -4,3 +4,5 @@
 
 
 ## Features
+
+* Add support for Server Side Encryption with s3 for RGW.
```

Comment:
```suggestion
* Add support for Amazon Server Side Encryption with s3 for RGW.
```

Also add a link to https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html?